lib/radiator/directory/editor/permission.ex

```diff
@@ -25,6 +26,9 @@ defmodule Radiator.Directory.Editor.Permission do
   def get_permission(user = %Auth.User{}, subject = %Audio{}),
     do: do_get_permission(user, subject)
 
+  def get_permission(_user = %Auth.User{}, _subject = %Chapter{}),
+    do: nil
+
```

Comment:
Hmm - this is kind of wonky. Since we don't have any permissions on Chapters (and we don't want to have them), I think it is more appropriate to reference the audio in the callsite, rather than do this monkeypatching here.

E.g. all callsites should go:
```elixir
has_permission(actor. %Audio{id: chapter.audio_id}, :edit)
```

instead. If you insist to expose the Chapter here, then make it a short circuit of the has_permission
```elixir
def has_permission(actor = %Auth.User{}, _subject = %Chapter{audio_id: audio_id},permission), do: has_permission(actor, %Audio{id: audio_id}, permission)
```